staging/knative/charts/serving/files/crds.yaml

```diff
@@ -395,9 +414,11 @@ spec:
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
+  annotations:
+    helm.sh/hook: crd-install
+    helm.sh/hook-delete-policy: before-hook-creation
```

Comment:
`helm.sh/hook-delete-policy: before-hook-creation` these are not recommended. These will delete any resources whenever they are run that are of the types defined by the CRD. I would remove these.

Reply:
Thanks for pointing that out. Removed.